R/utils.R

```diff
@@ -61,44 +59,44 @@ parse_generic = function(api_answer, ...) {
 #' Returns data.frame from parsed xml API answer.
 #'
 #' @inheritParams parse_publisher
-#' @param multiple {`logical(1)`}\cr{}
+#' @param multiple \[`logical(1)`\]\cr{}
 #'                 If multiple results match your query, should the function
 #'                 recursively fetch data for each of one of them
 #'                 (`multiple = TRUE`) or return a data.frame containing only
 #'                 titles and ISSN of all matches (`multiple = FALSE`)
 #' @inheritParams check_key
 #'
 #' @return Returns a data.frame with the following columns:
-#' * `title`        {`character(1)`}\cr{}
+#' * `title`        \[`character(1)`\]\cr{}
 #'                  the name of the journal
-#' * `issn`         {`character(1)`}\cr{}
+#' * `issn`         \[`character(1)`\]\cr{}
 #'                  the ISSN of the journal
-#' * `romeocolour`  {`character(1)`}\cr{}
+#' * `romeocolour`  \[`character(1)`\]\cr{}
 #'                  the SHERPA/RoMEO colour of the journal
-#' * `preprint`     {`character(1)`}\cr{}
+#' * `preprint`     \[`character(1)`\]\cr{}
 #'                  is the preprint (not reviewed) archivable?
-#' * `postprint`    {`character(1)`}\cr{}
+#' * `postprint`    \[`character(1)`\]\cr{}
 #'                  is the postprint (reviewed but not formatted) archivable?
-#' * `pdf`          {`character(1)`}\cr{}
+#' * `pdf`          \[`character(1)`\]\cr{}
 #'                  is the publisher's version (reviewed and formatted)
-#' * `pre_embargo`  {`character(1)`}\cr{}
+#' * `pre_embargo`  \[`character(1)`\]\cr{}
 #'                  if applicable the embargo period before the author(s) can
 #'                  archive the preprint
-#' * `post_embargo` {`character(1)`}\cr{}
+#' * `post_embargo` \[`character(1)`\]\cr{}
 #'                  if applicable the embargo period before the author(s) can
 #'                  archive the postprint
-#' * `pdf_embargo`  {`character(1)`}\cr{}
+#' * `pdf_embargo`  \[`character(1)`\]\cr{}
 #'                  if applicable the embargo period before the author(s) can
 #'                  archive the publisher's version
 #'
 #' @keywords internal
 #'
 #' @importFrom httr content
 #' @importFrom xml2 xml_text xml_find_all xml_find_first
-parse_journal = function(xml_source, outcome, hits, multiple = FALSE,
-                         key = NULL) {
+parse_journal <- function(xml_source, outcome, hits, multiple = FALSE,
+                          key = NULL) {
 
-  if (outcome %in% c("singleJournal", "uniqueZetoc")) {
+  if (outcome %in% c("singleJournal", "uniqueZetoc") & multiple == TRUE) {
```

Comment:
`bool == TRUE` :thinking: 
```suggestion
  if (outcome %in% c("singleJournal", "uniqueZetoc") & multiple) {
```